### ISSUE TITLE
Is the project alive?

### ISSUE THREAD

**Issue by ottopic** [minhnc/TiDigits#1]
Hello,
I'm very interested in a module of Fabric Digit authentication tool. What about this project?

**Comment by minhnc** [comment]
No.. I used AccountKit instead. Here is Titanium module - https://github.com/hansemannn/ti.accountkit
